clang/lib/Driver/Multilib.cpp

```diff
@@ -124,12 +124,11 @@ bool MultilibSet::select(const Driver &D, const Multilib::flags_list &Flags,
     }
 
     // If this multilib is actually a placeholder containing a fatal
-    // error message written by the multilib.yaml author, display that
-    // error message, and return failure.
-    if (M.isFatalError()) {
-      D.Diag(clang::diag::err_drv_multilib_custom_error) << M.getFatalError();
-      return false;
-    }
+    // error message written by the multilib.yaml author, then set a
```

Comment:
On the line above, should that be "containing an error message written ... " rather than fatal error?